[2025-01-25T22:03:10.070+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-01-25T22:03:10.085+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: pull_gazettes_data.upload_contents_to_s3_task manual__2025-01-25T21:19:33.960892+00:00 [queued]>
[2025-01-25T22:03:10.094+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: pull_gazettes_data.upload_contents_to_s3_task manual__2025-01-25T21:19:33.960892+00:00 [queued]>
[2025-01-25T22:03:10.094+0000] {taskinstance.py:2865} INFO - Starting attempt 10 of 11
[2025-01-25T22:03:10.107+0000] {taskinstance.py:2888} INFO - Executing <Task(UploadContentsToS3): upload_contents_to_s3_task> on 2025-01-25 21:19:33.960892+00:00
[2025-01-25T22:03:10.121+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=29468) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-01-25T22:03:10.122+0000] {standard_task_runner.py:72} INFO - Started process 29481 to run task
[2025-01-25T22:03:10.122+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'pull_gazettes_data', 'upload_contents_to_s3_task', 'manual__2025-01-25T21:19:33.960892+00:00', '--job-id', '199', '--raw', '--subdir', 'DAGS_FOLDER/dag_script.py', '--cfg-path', '/tmp/tmpbnd7afbr']
[2025-01-25T22:03:10.124+0000] {standard_task_runner.py:105} INFO - Job 199: Subtask upload_contents_to_s3_task
[2025-01-25T22:03:10.162+0000] {task_command.py:467} INFO - Running <TaskInstance: pull_gazettes_data.upload_contents_to_s3_task manual__2025-01-25T21:19:33.960892+00:00 [running]> on host 41bcec3d818b
[2025-01-25T22:03:10.232+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='***@example.com' AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='pull_gazettes_data' AIRFLOW_CTX_TASK_ID='upload_contents_to_s3_task' AIRFLOW_CTX_EXECUTION_DATE='2025-01-25T21:19:33.960892+00:00' AIRFLOW_CTX_TRY_NUMBER='10' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-01-25T21:19:33.960892+00:00'
[2025-01-25T22:03:10.234+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-01-25T22:03:10.253+0000] {base.py:84} INFO - Retrieving connection 'minio_s3_conn_id'
[2025-01-25T22:03:10.254+0000] {connection_wrapper.py:391} INFO - AWS Connection (conn_id='minio_s3_conn_id', conn_type='aws') credentials retrieved from extra.
[2025-01-25T22:03:10.255+0000] {warnings.py:112} WARNING - <string>:9: AirflowProviderDeprecationWarning: extra['host'] is deprecated and will be removed in a future release. Please set extra['endpoint_url'] instead

[2025-01-25T22:03:10.490+0000] {logging_mixin.py:190} INFO - Loading object /usr/local/***/data/ingestion_checkpoint_logs into S3 bucket diario-oficial-db-dump/ingestion_checkpoint_logs/
[2025-01-25T22:03:10.492+0000] {logging_mixin.py:190} INFO - Loading folder /usr/local/***/data/ingestion_checkpoint_logs into S3 bucket diario-oficial-db-dump/ingestion_checkpoint_logs/
[2025-01-25T22:03:10.649+0000] {logging_mixin.py:190} INFO - Loading object /usr/local/***/data/texts into S3 bucket diario-oficial-db-dump/texts/
[2025-01-25T22:03:10.651+0000] {logging_mixin.py:190} INFO - Loading folder /usr/local/***/data/texts into S3 bucket diario-oficial-db-dump/texts/
[2025-01-25T22:03:10.824+0000] {logging_mixin.py:190} INFO - Loading object /usr/local/***/data/preprocessed_documents/ into S3 bucket diario-oficial-db-dump/preprocessed_documents
[2025-01-25T22:03:10.826+0000] {logging_mixin.py:190} INFO - Loading folder /usr/local/***/data/preprocessed_documents/ into S3 bucket diario-oficial-db-dump/preprocessed_documents
[2025-01-25T22:03:12.061+0000] {logging_mixin.py:190} INFO - Loading object /usr/local/***/data/vector_db/ into S3 bucket diario-oficial-db-dump/vector_db/
[2025-01-25T22:03:12.063+0000] {logging_mixin.py:190} INFO - Loading folder /usr/local/***/data/vector_db/ into S3 bucket diario-oficial-db-dump/vector_db/
[2025-01-25T22:03:12.223+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-01-25T22:03:12.223+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=pull_gazettes_data, task_id=upload_contents_to_s3_task, run_id=manual__2025-01-25T21:19:33.960892+00:00, execution_date=20250125T211933, start_date=20250125T220310, end_date=20250125T220312
[2025-01-25T22:03:12.264+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-01-25T22:03:12.279+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-01-25T22:03:12.280+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
